ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -21,6 +21,7 @@ Create Pipeline Server
     ...                It assumes the Data Connection is aleady created
     ...                and you wants to use defaul DB configurations [TEMPORARY]
     [Arguments]    ${dc_name}    ${project_title}
+    ...            ${aws_s3_ca_bundle}=${NONE}
```

Comment:
I think just ${s3_ca_bundle} will be more generic, as we will probably use this with minio.

I'm not sure if the right default value should be ${NONE} or ${EMPTY} (we have both options used in ods-ci a lot)

Reply:
Even if we use the minio, we have to set the same AWS variables.

If it is  ${NONE} or ${EMPTY} it doesn't matter in the implementation.